Update SharedArrayBuffer article (#653)
GoogleChrome/developer.chrome.com: site/en/blog/enabling-shared-array-buffer/index.md

- `SharedArrayBuffer` is currently available in Desktop Chrome, but from Chrome
  91 it will be limited to cross-origin isolated pages. If you don't think you
  can make this change in time, you can [register for an origin trial](#origin-trial) to retain
  the current behavior until at least Chrome 96.
- If you intend to enable cross-origin isolation to continue using
  `SharedArrayBuffer` evaluate the impact this will have on other cross-origin

>   elements on your website, e.g. ad placements. Check if `SharedArrayBuffer` is

Reviewer: ```suggestion
  elements on your website, such as ad placements. Check if `SharedArrayBuffer` is
```

`Cross-Origin-Embedder-Policy` and `Cross-Origin-Opener-Policy`.

If you don't think you can make these changes in time for Chrome 91, you can
[register for an origin trial](#origin-trial) to retain current Desktop Chrome
behavior until at least Chrome 96.

{% Aside %}

> **Update on April 2021**

Reviewer: ```suggestion
**Update, April 2021**
```

We've been exploring ways to deploy `Cross-Origin-Resource-Policy` at scale, as
cross-origin isolation requires all subresources to explicitly opt-in. And we
have come up with the idea of going in the opposite direction: a new [COEP
"credentialless" mode](https://github.com/mikewest/credentiallessness/) that
allows loading resources without the CORP header by stripping all their

> credentials. We are working out how it should work in detail, but we hope this
> will lighten your burden of making sure the subresources are sending the
> `Cross-Origin-Resource-Policy` header.

Reviewer: ```suggestion
credentials. We are figuring out the details of how it should work, but we hope this
will lighten your burden of making sure the subresources are sending the
`Cross-Origin-Resource-Policy` header.
```

Also, it's known that the `Cross-Origin-Opener-Policy: same-origin` header will
break integrations that require cross-origin window interactions such as OAuth
and payments. To mitigate this problem, we are [exploring relaxing the
condition](https://github.com/whatwg/html/issues/6364) to enable cross-origin
isolation to `Cross-Origin-Opener-Policy: same-origin-allow-popups`. This way
the communication with the window opened by itself will be possible.

If you want to enable cross-origin isolation to use `SharedArrayBuffer` but are
blocked by these challenges, we recommend [registering for an origin

> trial](#origin-trial) and wait until the new modes are available. We are not

Reviewer: ```suggestion
trial](#origin-trial) and waiting until the new modes are available. We are not
```
